src/pages/Map/components/FocusedLocationBottomSheet/FocusedLocationBottomSheet.module.css

```
@@ -1,6 +1,7 @@
 .DetailInfoBottomSheetContainer {
   position: relative;
   width: 100%;
+  max-width: var(--custom-max-width);
   position: absolute;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

position is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)